starting_package/openvino_workbench/arguments_parser.py

```diff
@@ -18,203 +18,246 @@
+                                  default='workbench')
+
+        # Misc
+        self._parser.add_argument('--base-prefix',
+                                  required=False,
+                                  help='Specifies the base prefix of the DL Workbench web application.',
+                                  default='/')
+
+    def _validate_arguments(self):
+        self._validate_restart_arguments()
+        self._validate_ssl_arguments()
+        if platform.system() == 'Windows':
+            self._validate_arguments_for_windows()
+
+    def _validate_ssl_arguments(self):
+        self._logger.debug('Validating arguments for SSL.')
+        # Check for SSL-related files
+        if not self.arguments.assets_directory and (self.arguments.ssl_key_name or self.arguments.ssl_certificate_name):
+            self._logger.debug('SSL key and/or certificate were provided without the assets directory.')
+            self._parser.error('ERROR: "--assets-directory" is required for SSL. SSL key and certificate should be '
+                               'placed there.'
+                               '\nExample command:'
+                               f'\n\n\t{CLI_COMMAND} --assets-directory /path/to/assets --ssl-key-name key.pem '
+                               '--ssl-certificate-name certificate.pem'
+                               f'{ABORTING_EXIT_MESSAGE}')
+        if (not self.arguments.ssl_key_name and self.arguments.ssl_certificate_name) or (
+                self.arguments.ssl_key_name and not self.arguments.ssl_certificate_name):
+            self._logger.debug('Only one of the SSL files was provided.')
+            self._parser.error('ERROR: Both SSL certificate name and SSL key name are required.'
+                               '\nExample command:'
+                               f'\n\n\t{CLI_COMMAND} --assets-directory /path/to/assets --ssl-key-name key.pem '
+                               '--ssl-certificate-name certificate.pem'
+                               f'{ABORTING_EXIT_MESSAGE}')
+
+    def _validate_restart_arguments(self):
+        if not self.arguments.restart:
+            return
+        self._logger.debug('Validating arguments for restart.')
+        # If detached restart is needed then there should be exactly 4 arguments
+        # and '--detached' should be one of them
+        if self.arguments.detached and len(sys.argv) != 4:
+            self._logger.debug('Error with restarting in the detached mode. Incorrect number of arguments. '
+                               f'{vars(self.arguments)}')
+            self._parser.error(
+                'ERROR: Unrecognized arguments for restart. '
+                'To restart the container in the detached mode, '
+                'provide the "--detached" argument and the container name following the "--restart" argument. '
+                'The only other argument available with "--restart" is "--detached".'
```

Comment:
Feels like we've had this conversation already, but this says we need 4 arguments, and I see only -restart and - detached